libs/dh/wholesale/feature-settlement-reports/src/lib/modal/dh-request-settlement-report-modal.component.html

```diff
@@ -94,6 +94,12 @@
           }
         </div>
       }
+
+      <ng-container *dhFeatureFlag="'settlement-report-use-api'">
```

Comment:
Question: Why not use the directive on checkbox component?

Reply:
Even better. Fixed :)